composer.json

```diff
@@ -23,6 +23,7 @@
         }
     },
     "require": {
+        "php": "^8.0",
         "statamic/cms": "^3.0"
```

Comment:
@ryanmitchell as you bumped the tests to Laravel 9, perhaps we should do Statamic 3.3 only? What do you think @jasonvarga ?

Reply:
Yup. Shoulda mentioned that myself. 👍